Makefile

```diff
@@ -369,15 +363,15 @@ ut combined.coverprofile: $(SRC_FILES)
 ###############################################################################
 # CI/CD
 ###############################################################################
+.PHONY: mod-download
+mod-download:
+	-$(DOCKER_RUN) $(CALICO_BUILD) go mod download
```

Comment:
doesn't this preceding dash ignore failures? Isn't that undesirable?

Reply:
Yes, the idea with `mod-downlaod` is so the actual builds can retry any failed downloads. Sort of a cache warmup stage.